mitodl/bootcamp-ecommerce
# Added user application dashboard (list view)

#### What are the relevant tickets?
Closes #523 

#### What's this PR do?
Adds a dashboard for user applications (list view only)

#### How should this be manually tested?
Add some applications for your user and log in. You should see you applications listed and the body of the page should match the designs

#### Any background context you want to provide?
https://projects.invisionapp.com/d/main#/console/19928472/417192809/preview

#### Screenshots (if appropriate)
![ss 2020-05-27 at 15 36 58 ](https://user-images.githubusercontent.com/14932219/83145949-92316900-a0c3-11ea-8d6a-dd4b553182a9.png)
![ss 2020-05-27 at 15 36 36 ](https://user-images.githubusercontent.com/14932219/83145952-93629600-a0c3-11ea-9259-a7fa0f0d1ed6.png)
![ss 2020-05-27 at 15 36 26 ](https://user-images.githubusercontent.com/14932219/83145956-93fb2c80-a0c3-11ea-9aa8-fc4969cc4932.png)
![ss 2020-05-27 at 15 35 43 ](https://user-images.githubusercontent.com/14932219/83145960-9493c300-a0c3-11ea-8d07-5250b97a836a.png)
